packages/SwingSet/src/liveslots/vatstore-usage.md

```diff
@@ -117,45 +119,46 @@ Each function accepts an `isDurable` argument, so there are currently 8 collecti
+The interpretation of a vref therefore varies based on whether the initial "type" portion before a slash (`o+${exportID}`) identifies a collection type or a virtual object kind:
 
-* `o+11/1` : `o+11` is a kind, so `/1` refers to the first instance of that kind
-* `o+11/2` : second instance of that kind
-*
-* `o+6/1` : `o+6` is a collection type (scalarDurableMapStore), so `/1` refers to the first collection in the vat
-* `o+7/2` : second collection in the vat, which happens to be of type `o+7` (scalarDurableWeakMapStore)
-* `o+5/3` : third collection in the vat, of type `o+5` (scalarWeakSetStore)
-* `o+5/4` : fourth collection in the vat, also a scalarWeakSetStore
+* `o+6/1`: `6` is a collection type in the storeKindIDTable ("scalarDurableMapStore"), so `o+6/1` refers to the first collection (of any type) in the vat
+* `o+7/1`: `7` is also a collection type in the storeKindIDTable ("scalarDurableWeakMapStore"), so `o+7/1` is guaranteed not to coexist with `o+6/1`
+* `o+7/2`: refers to the second collection (of any type) in the vat, which has type `7` ("scalarDurableWeakMapStore")
+* `o+5/3`: refers to the third collection (of any type) in the vat, which has type `5` ("scalarWeakSetStore")
+* `o+5/4`: refers to the fourth collection (of any type) in the vat, also a scalarWeakSetStore
+* `o+11/1`: `11` is not a collection type in the storeKindIDTable, so is instead a kind and `o+11/1` refers to the first instance of that kind
+* `o+11/2`: refers to the second instance of that kind
 
 
 # Baggage
 
 Most collections are created by userspace code, but to support vat upgrade, liveslots creates one special collection named "baggage". This is a `scalarDurableMapStore` that is passed into the third argument of `buildRootObject`.
 
-This object needs to be pre-generated because the second (and subsequent) versions of the vat will use it to reach all other durable objects from their predecessors, so v2 can remember things that were stored by v1. The most significant values of "baggage" are the KindHandles for durable Kinds made by v1. V2 will need these to call `defineDurableKind` and re-attach behavior for each one. Each version is obligated to re-attach behavior for *all* durable Kinds created by their predecessor, to satisfy the obligates created when the older version exported durable objects of those Kinds.
+This object needs to be pre-generated because the second (and subsequent) versions of the vat will use it to reach all other durable objects from their predecessors, so v2 can remember things that were stored by v1. The most significant values of "baggage" are the KindHandles for durable Kinds made by v1. V2 will need these to call `defineDurableKind` and re-attach behavior for each one. Each version must re-attach behavior for *all* durable Kinds created by its predecessors, to satisfy the obligations created when the older version exported durable objects of those Kinds.
```

Comment:
good cleanup, maybe add a "note: the `defineKind()` is what creates the obligation, even if no instances of that Kind are ever created"